.circleci/config.yml

```diff
@@ -22,8 +22,7 @@ jobs:
       - run:
           name: install python dependencies
           command: |
-            curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_VERSION=1.1.8 python
-            . $HOME/.poetry/env
```

Comment:
Thanks @victoriachan your suggestion worked. I just needed to remove this line so it's not sourcing a directory that no longer exists.

Reply:
Great!